schedule_lessons/dashboard/views.py

```diff
@@ -582,10 +587,10 @@ def change_password(request):
             'status_code': 400
         }
         current_user = request.user
-        if current_user.social_auth.filter(provider='google-oauth2'):
-            data['social_error'] = ('You are using a google account so '
-                                    "you can't change your password.")
-            return JsonResponse(data)
+        # if current_user.social_auth.filter(provider='google-oauth2'):
```

Comment:
do we still need this section? or was it commented for testing and then meant to be removed?